src/Logger/AuditLogger.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Logger;
+
+use Psr\Log\LoggerInterface;
+use SumoCoders\FrameworkCoreBundle\Enum\EventAction;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+class AuditLogger
+{
+    public function __construct(
+        private readonly Security $security,
+        private readonly RequestStack $requestStack,
+        private readonly LoggerInterface $auditTrailLogger,
+    ) {
+    }
+
+    public function log(
+        ?string $entityClass = null,
+        ?string $identifier = null,
+        EventAction $action = EventAction::READ,
```

Comment:
Wat is een read actie?

Reply:
Bv het uitlezen van een entity, dat kan optioneel handmatig worden gelogd. Is voor waneer je wilt weten wie een bepaalde dataset heeft gezien